javascript/implementation/MemoryStore.ts

```diff
@@ -486,46 +429,65 @@ export class MemoryStore implements Store {
      * @param asOf show results as of a time in the past
      * @returns a promise of a list of ChangePairs
      */
-    async getOrderedEntries(container: Muid, through = Infinity, asOf?: AsOf): Promise<Entry[]> {
-        const asOfTs: Timestamp = asOf ? (this.asOfToTimestamp(asOf)) : generateTimestamp();
-        const containerId: [number, number, number] = [container?.timestamp ?? 0, container?.medallion ?? 0, container?.offset ?? 0];
-        const lower = this.entriesByContainerKeyPlacement.lowerBound(muidTupleToString(containerId));
-        const upper = this.entriesByContainerKeyPlacement.upperBound(`${muidTupleToString(containerId)},${asOfTs}`);
+    async getOrderedEntries(container: Muid, through = Infinity, asOf?: AsOf):
+        Promise<Map<string, Entry>> {
+        return Promise.resolve(this.getOrderedEntriesHelper(container, through, asOf));
+    }
 
-        let clearanceTime: Timestamp = this.getLastClearanceTimeForContainer(muidTupleToString(containerId), asOfTs);
-        const returning = <Entry[]>[];
+    getOrderedEntriesHelper(container: Muid, through = Infinity, asOf?: AsOf): Map<string, Entry> {
```

Comment:
Should the helper methods (keyed entries and ordered entries) be private?

Reply:
it's useful for debugging and testing if they're accessible.  Since they're not part of the public Store interface though I'm not worried about people using them without knowing what they're doing.